packages/models-library/src/models_library/api_schemas_resource_usage_tracker/licensed_items_checkouts.py

```diff
@@ -15,6 +15,8 @@
 class LicensedItemCheckoutGet(BaseModel):
     licensed_item_checkout_id: LicensedItemCheckoutID
     licensed_item_id: LicensedItemID
+    key: str
+    version: str
```

Comment:
to reduce confusion here, I would suggest defining at least a type alias such as `LicensedItemKey` and `LicensedItemVersion`